src/opnsense/scripts/openvpn/ovpn_service_control.php

```diff
@@ -0,0 +1,183 @@
+#!/usr/local/bin/php
```

Comment:
this script looks a bit convoluted (new/different start,stop,etc) but you probably meant to extract bootstrap/setup code from legacy side

Reply:
yes, I don't want to mangle the existing hooks as this will likely lead to more noise and the "configure" action is required to cleanly apply pending changes (adding the start/[re]start doesn't increase the boilerplate)